src/Components/Task-editor.tsx

```diff
@@ -0,0 +1,63 @@
+import { inject, observer } from "mobx-react";
+import React from "react";
+import TaskItem from "../Models/TaskItem";
+import TaskStore from "../Stores/TaskStore";
+
+@inject('TaskStore')
+@observer
+export default class TaskEditor extends React.Component<{ taskStore: typeof TaskStore }> {
+    textTask: any;
+
+    addTodo = (arg0: { title: string; completed: boolean; }) => {
```

Comment:
better to avoid `arg0`  like naming conventions 
`{ title: string; completed: boolean; }` here better to add these in to types or interface

Reply:
This is fixed . I have used direct object